src/Components/Web.JS/src/Rendering/BrowserRenderer.ts

```diff
@@ -515,18 +553,20 @@ function stripOnPrefix(attributeName: string) {
   throw new Error(`Attribute should be an event name, but doesn't start with 'on'. Value: '${attributeName}'`);
 }
 
-function setDeferredElementValue(element: Element, value: string | null) {
-  if (element instanceof HTMLSelectElement) {
-    // There's no sensible way to represent a select option with value 'null', because
-    // (1) HTML attributes can't have null values - the closest equivalent is absence of the attribute
-    // (2) When picking an <option> with no 'value' attribute, the browser treats the value as being the
-    //     *text content* on that <option> element. Trying to suppress that default behavior would involve
-    //     a long chain of special-case hacks, as well as being breaking vs 3.x.
-    // So, the most plausible 'null' equivalent is an empty string. It's unfortunate that people can't
-    // write <option value=@someNullVariable>, and that we can never distinguish between null and empty
-    // string in a bound <select>, but that's a limit in the representational power of HTML.
-    element.value = value || '';
-  } else {
-    (element as any).value = value;
+function setSingleSelectElementValue(element: HTMLSelectElement, value: string | null) {
+  // There's no sensible way to represent a select option with value 'null', because
+  // (1) HTML attributes can't have null values - the closest equivalent is absence of the attribute
+  // (2) When picking an <option> with no 'value' attribute, the browser treats the value as being the
+  //     *text content* on that <option> element. Trying to suppress that default behavior would involve
+  //     a long chain of special-case hacks, as well as being breaking vs 3.x.
+  // So, the most plausible 'null' equivalent is an empty string. It's unfortunate that people can't
+  // write <option value=@someNullVariable>, and that we can never distinguish between null and empty
+  // string in a bound <select>, but that's a limit in the representational power of HTML.
+  element.value = value || '';
+}
+
+function setMultipleSelectElementValue(element: HTMLSelectElement, value: string[]) {
+  for (let i = 0; i < element.options.length; i++) {
+    element.options[i].selected = value.indexOf(element.options[i].value) !== -1;
```

Comment:
I think I must be missing some understanding here. It looks like you're removing the concept of setting deferred values for everything except *select* elements. Doesn't this break the previous work you did for `input`?

Reply:
These functions just set the `value` attribute of the element. The `_blazorDeferredElementValue` property is set inside `tryApplyValueProperty`. `setDeferredElementValue` set the `value` attribute of an element whose `value` property was set in a deferred manner, rather than set the `_blazorDeferredElementValue` property of such elements.